AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/EditTaskCommandHandler.cs

```diff
@@ -56,16 +56,25 @@ public EditTaskCommandHandler(AllReadyContext context)
             }
 
             // Add new attachment
-            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.Name))
+            if (message.Task.NewAttachment != null && !string.IsNullOrEmpty(message.Task.NewAttachment.FileName))
```

Comment:
Aren't we already doing this validation in `TaskEditViewModelValidator`? If so, then we only want to validate incoming data once, and that's as close to the UI side as we can. If we're not doing this validation in `TaskEditViewModelValidator`,  then please update this to:
`if (!string.IsNullOrEmpty(message.Task.NewAttachment?.FileName))`